android/app/src/main/java/io/locha/p2p/service/ChatServiceModule.java

```diff
@@ -209,6 +215,37 @@ public String getName() {
         promise.resolve(this.isServiceStarted && Runtime.isStarted());
     }
 
+
+
+    public void getExternalAddressWithTime()  {
+
+         Thread thread = new Thread(new Runnable() {
+             public void run() {
+                 try{
+                     Thread.sleep(2000);
+
+                     Timer timer = new Timer();
+
+                     timer.schedule( new TimerTask() {
+                         public void run() {
+                             String[] ip =  Runtime.getInstance().externalAddresses();
+
+                             for (String i : ip) {
+                                 event.onExternalAddress(i);
+                             }
+                         }
+                     }, 0, 60000);
```

Comment:
Add a constant for `60000`

Reply:
is it necessary?

Reply:
Yep, a constant expresses the intention of the value and it's meaning, could be nicer with a comment with the value explained in minutes or hours.